crescent/internal/handle_resp.py

```diff
@@ -39,27 +39,28 @@
 __all__: Sequence[str] = ("handle_resp",)
 
 
+active_clients: dict[Any, Client] = {}
```

Comment:
I think I see why you're doing this. Would it be better to make it `dict[int, Client]` and then make the keys `id(app)` rather than `app`? I have no idea what the hash for a hikari bot is.

Reply:
Even if the hash for a hikari bot is fine, not using Any seems like reason enough to use id(), especially since you don't actually need the *value* of the key (you already have it, since you're indexing by it)

Reply:
You're right. I can't rely on the hash cause some random impl might not even support that.